bin/setup

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+
+sh ./bin/composer install --no-scripts
```

Comment:
Why are we running these with `sh` when they've got hashbangs that have them run with `bash`?

Reply:
Sometimes we've lost the +x file mod on those for random reasons

Reply:
that'll probably be me messing up permissions due to committing both in WSL & Windows 😅 

I'm fine with not relying on the hashbang, but think we should use `bash` instead of `sh`.

Reply:
I believe that using sh will comply with what shell requested after the hashbang, or am I understanding something different?

Reply:
No - the hashbang is a shell thing: when executing a file directly, it checks the first line of the file for a hashbang and if it finds it, uses the program it points to to run that file.

When you pass a script directly to `sh`, it executes that script as a shell script, regardless of its hashbang:

```
/c/Users/G-Rath
❯ head script.js
#!/usr/bin/env node

const stdTimezoneOffset = (date) => {
    const jan = new Date(date.getFullYear(), 0, 1);
    const jul = new Date(date.getFullYear(), 6, 1);
    return Math.max(jan.getTimezoneOffset(), jul.getTimezoneOffset());
}

const isDstObserved = (date) => {
    const jan = new Date(date.getFullYear(), 0, 1);

/c/Users/G-Rath
❯ sh script.js
script.js: 3: script.js: Syntax error: "(" unexpected
```

Reply:
huh

Reply:
done